ironfish/src/rpc/routes/wallet/types.ts

```diff
@@ -134,7 +133,14 @@ export const RpcWalletTransactionSchema: yup.ObjectSchema<RpcWalletTransaction>
   })
   .defined()
 
-export type RpcAccountImport = Omit<AccountImport, 'createdAt'> & {
+export type RpcAccountImport = {
```

Comment:
just to keep things consistent. we can do this in other places, but explicitly defining the interface here like we do everywhere else